# Russian translations for Django package.
# Copyright (C) Django Software Foundation and contributors
# This file is distributed under the same license as the Django package.
# Automatically generated, 2013.
# Tyshchenko Viktor <tyvik8@gmail.com>, 2014, 2015.
#
msgid ""
msgstr ""
"Project-Id-Version: Django 1.6\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-04-05 11:28+0300\n"
"PO-Revision-Date: 2015-03-31 15:42+0300\n"
"Last-Translator: Tyshchenko Viktor <tyvik8@gmail.com>\n"
"Language-Team: none\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Gtranslator 2.91.6\n"

# 798b947859e044c9997de6a0b6edd5e0
#: ../../faq/troubleshooting.txt:3
msgid "Troubleshooting"
msgstr "Поиск и устранение неисправностей"

# 4f9b3b7054a84197bfe71547ab0f43bc
#: ../../faq/troubleshooting.txt:5
msgid ""
"This page contains some advice about errors and problems commonly "
"encountered during the development of Django applications."
msgstr ""
"Эта страница содержит несколько советов по поводу ошибок и проблем, обычно "
"возникающих в процессе разработки приложений на Django."

# fb0dcd5c6349424c9b40b35a022d1a47
#: ../../faq/troubleshooting.txt:11
#, fuzzy
msgid "Problems running ``django-admin``"
msgstr "Трудности с запуском django-admin"

# 4b08f7db18104b39bd083ab48bedc572
#: ../../faq/troubleshooting.txt:14
#, fuzzy
msgid "\"command not found: `django-admin`\""
msgstr "\"command not found: django-admin\""

# e335ed32b57a436eb85bdc480ddf6055
#: ../../faq/troubleshooting.txt:16
msgid ""
":doc:`django-admin </ref/django-admin>` should be on your system path if you "
"installed Django via ``python setup.py``. If it's not on your path, you can "
"find it in ``site-packages/django/bin``, where ``site-packages`` is a "
"directory within your Python installation. Consider symlinking to :doc:"
"`django-admin </ref/django-admin>` from some place on your path, such as :"
"file:`/usr/local/bin`."
msgstr ""
":doc:`django-admin </ref/django-admin>` должен уже быть доступным по путям, "
"если вы устанавливали Django с помощью ``python setup.py``. В противном "
"случае этот скрипт можно найти в ``site-packages/django/bin``, где ``site-"
"packages`` - каталог с установленным Python. Настоятельно советуем сделать "
"символическую ссылку :doc:`django-admin </ref/django-admin>` куда-нибудь в :"
"file:`/usr/local/bin`."

#: ../../faq/troubleshooting.txt:23
msgid ""
"If ``django-admin`` doesn't work but ``django-admin.py`` does, you're "
"probably using a version of Django that doesn't match the version of this "
"documentation. ``django-admin`` is new in Django 1.7."
msgstr ""
"Если ``django-admin`` не работает, а ``django-admin.py`` работает, скорее "
"всего вы используете версию Django, которая не соответствует этой "
"документации. ``django-admin`` была добавлена Django 1.7."

# d0c31f4e944e42c9a6f890e226d354bb
#: ../../faq/troubleshooting.txt:28
msgid "Mac OS X permissions"
msgstr "Права доступа на Mac OS X"

# 6fd18eeb8fe845e0a341f7fac1f6bb84
#: ../../faq/troubleshooting.txt:30
msgid ""
"If you're using Mac OS X, you may see the message \"permission denied\" when "
"you try to run ``django-admin``. This is because, on Unix-based systems like "
"OS X, a file must be marked as \"executable\" before it can be run as a "
"program. To do this, open Terminal.app and navigate (using the ``cd`` "
"command) to the directory where :doc:`django-admin </ref/django-admin>` is "
"installed, then run the command ``sudo chmod +x django-admin``."
msgstr ""
"Если у вас Mac OS X, то при запуске ``django-admin`` можете получить "
"сообщение \"permission denied\". Это происходит потому что в Unix-подобных "
"системах (например, OS X) надо явно указать скрипту, что он является "
"исполняемым. Чтобы это сделать откройте терминал и перейдите (с помощью "
"команды ``cd``) в каталог, где установлен :doc:`django-admin </ref/django-"
"admin>`, затем выполните ``sudo chmod +x django-admin``."

# 4733e0f05a484830b656a1e3e36d8a40
#: ../../faq/troubleshooting.txt:38
msgid "Miscellaneous"
msgstr "Разное"

# 05c5d85697814707ad5d475f25900c3a
#: ../../faq/troubleshooting.txt:41
msgid "I'm getting a ``UnicodeDecodeError``. What am I doing wrong?"
msgstr "Я получил ``UnicodeDecodeError``. Что я делаю не так?"

# 2ee2eb598b8848b4a0eb0813bcc416de
#: ../../faq/troubleshooting.txt:43
msgid ""
"This class of errors happen when a bytestring containing non-ASCII sequences "
"is transformed into a Unicode string and the specified encoding is "
"incorrect. The output generally looks like this::"
msgstr ""
"Такую ошибку можно получить, если байтовую строку с не-ASCII символами "
"преобразовывать в Unicode строку, используя не правильную кодировку. Обычно "
"ошибка выглядит следующим образом::"

# 81ff8883e38c4876bb500d53826158b0
#: ../../faq/troubleshooting.txt:50
msgid ""
"The resolution mostly depends on the context, however here are two common "
"pitfalls producing this error:"
msgstr ""
"Решение обычно зависит от контекста, но вот две распространенные причины "
"появления ошибки:"

# 415fae7d901a4a569511be05a5f922fa
#: ../../faq/troubleshooting.txt:53
msgid ""
"Your system locale may be a default ASCII locale, like the \"C\" locale on "
"UNIX-like systems (can be checked by the ``locale`` command). If it's the "
"case, please refer to your system documentation to learn how you can change "
"this to a UTF-8 locale."
msgstr ""
"Система по умолчанию использует ASCII локаль, такую как \"C\" в UNIX-"
"системах (можно проверить командой ``locale``). Если это так, ищите в "
"документации вашей ОС как изменить локаль на UTF-8."

# 3a7678922d59415694145fb7564431d2
#: ../../faq/troubleshooting.txt:58
msgid "You created raw bytestrings, which is easy to do on Python 2::"
msgstr "Вы создали байтовую строку, что очень просто в Python 2::"

# 72d1caaf584b4f698dd2fba8512564bf
#: ../../faq/troubleshooting.txt:62
msgid ""
"Either use the ``u''`` prefix or even better, add the ``from __future__ "
"import unicode_literals`` line at the top of your file so that your code "
"will be compatible with Python 3.2 which doesn't support the ``u''`` prefix."
msgstr ""
"используйте префикс ``u''``, или лучше добавьте ``from __future__ import "
"unicode_literals`` в начале файла и ваш код будет совместим с Python 3.2 без "
"использования перфикса ``u''``."

# 37bdf1ba358749e18d45e2430108b236
#: ../../faq/troubleshooting.txt:67
msgid "Related resources:"
msgstr "Близкие по теме ресурсы:"

# e27b76d32943413694a1a7a378d994c1
#: ../../faq/troubleshooting.txt:69
msgid ":doc:`Unicode in Django </ref/unicode>`"
msgstr ":doc:`Unicode в Django </ref/unicode>`"

# d14141851bed4c21afcf8e2b917bdac6
#: ../../faq/troubleshooting.txt:70
msgid "https://wiki.python.org/moin/UnicodeDecodeError"
msgstr "https://wiki.python.org/moin/UnicodeDecodeError"

# dba1caaf6c414b65b1ab309cd26f2f6e
#~ msgid "Script name may differ in distribution packages"
#~ msgstr "Имя скрипта может отличаться в разных пакетах"

# 52a739fbb8a84d09b10b70834b224c54
#~ msgid ""
#~ "If you installed Django using a Linux distribution's package manager (e."
#~ "g. ``apt-get`` or ``yum``) ``django-admin.py`` may have been renamed to "
#~ "``django-admin``; use that instead."
#~ msgstr ""
#~ "Если вы устанавливали Django через пакетный менеджер (``apt-get``, "
#~ "``yum`` и пр.), то ``django-admin.py`` может быть переименован в ``django-"
#~ "admin``."

# a28f01a2587b429487b467cdd842516f
#~ msgid "Running virtualenv on Windows"
#~ msgstr "Запуск виртуального окружения под Windows"

# 1d0977a0661c40b1a867216ee8b2e9c9
#~ msgid ""
#~ "If you used virtualenv_ to :ref:`install Django <installing-official-"
#~ "release>` on Windows, you may get an ``ImportError`` when you try to run "
#~ "``django-admin.py``. This is because Windows does not run the Python "
#~ "interpreter from your virtual environment unless you invoke it directly. "
#~ "Instead, prefix all commands that use .py files with ``python`` and use "
#~ "the full path to the file, like so: ``python C:\\pythonXY\\Scripts"
#~ "\\django-admin.py``."
#~ msgstr ""
#~ "Если вы использовали virtualenv_ для :ref:`установки Django <installing-"
#~ "official-release>` под Windows, то можете получить сообщение "
#~ "``ImportError`` при запуске ``django-admin.py``. Это происходит потому "
#~ "что Windows не запускает интерпретатор Python из виртуального окружения, "
#~ "пока вы явно не скажете ему это сделать. Для этого выполните ``python C:"
#~ "\\pythonXY\\Scripts\\django-admin.py``, не забудьте про расширение .py и "
#~ "используйте полный путь."
